### DISCUSSION TITLE
How to set traditional options

### DISCUSSION THREAD

**Discussion by w-r-z-k** [LunarVim/LunarVim#2057] [General]
I want to change to use relative numbers.

The documentation says:

```
To set options use

vim.opt.{option}
```

but where do I put this? Thanks.

**Comment by rebuilt** [comment]
Edit the configuration file found in ~/.config/lvim/config.lua.  

https://www.lunarvim.org/configuration/

**Comment by w-r-z-k** [comment]
It's not clear from the documentation where to configure non-lunar vim settings.

I have some understanding of how to configure neovim.  But lunarvim has its own settings files.

It's confusing.

**Comment by rebuilt** [comment]
Use config.lua for both Lunarvim and Neovim configuration options.  If you want to look at an example configuration you can find it here: https://github.com/abzcoding/lvim
